README.md

```diff
@@ -434,16 +434,12 @@ return p.count
 </tbody></table>
 
 El sobrecoste que tiene `defer` es extremadamente pequeño y sólo debe evitarse si puedes
-asegurar que asegurar que el tiempo de ejecución de tu función es de nanosegundos. Se gana mucho más con la legibilidad obtenida utilizando `defer` que el minúsculo coste que tiene utilizarlos. Esto se puede observar claramente en métodos largos que tienen más que simples accesos a memoria, dónde los otros cálculos son mucho más significantes que el `defer`.
+asegurar que el tiempo de ejecución de tu función es de nanosegundos. Se gana mucho más con la legibilidad obtenida utilizando `defer` que el minúsculo coste que tiene utilizarlos. Esto se puede observar claramente en métodos largos que tienen más que simples accesos a memoria, dónde los otros cálculos son mucho más significantes que el `defer`.
 
 ### El tamaño de los Canales es Uno o Ninguno
 
 Los canales normalmente deberían ser de tamaño uno o ser `unbuffered`. Por defecto, los canales son
-`unbuffered` y tienen un tamaño de cero. Cualquier otro tamaño debería de ser analizado con mucho detalle. Consider
-
-Channels should usually have a size of one or be unbuffered. By default,
-channels are unbuffered and have a size of zero. Any other size
-must be subject to a high level of scrutiny. Considerar el tamaño de nuestro canal evitará que acabemos sufriendo problemas de carga y bloqueos.
```

Comment:
Please, note that `Considerar el tamaño de nuestro canal evitará que acabemos sufriendo problemas de carga y bloqueos.` shouldn't be removed.